stetho-sample/src/debug/java/com/facebook/stetho/sample/APODContentProviderDumper.java

```diff
@@ -0,0 +1,112 @@
+package com.facebook.stetho.sample;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import com.facebook.stetho.dumpapp.DumpException;
+import com.facebook.stetho.dumpapp.DumperContext;
+import com.facebook.stetho.dumpapp.DumperPlugin;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class APODContentProviderDumper implements DumperPlugin {
```

Comment:
Prefer not to generalize this to content provider.  Instead, prefer `APODDumper`.

Separate nit: should be `*DumperPlugin` per convention.